renderdoc/data/glsl/mesh.geom

```diff
@@ -22,6 +22,12 @@
  * THE SOFTWARE.
  ******************************************************************************/
 
+#ifdef OPENGL_ES
+//#extension GL_EXT_geometry_shader : enable
+//#extension GL_OES_geometry_shader : enable
+//#extension GL_EXT_geometry_point_size : enable
```

Comment:
These `#extension` directives will be hoisted up and uncommented regardless of the `#ifdef`, because we do a simple string search. Since it's `enable` it will only give a warning, but this will break if someone innocently adds a `require`.

I think perhaps we should add `#extension_gles`, `#extension_opengl` and `#extension_vulkan` which can be conditionally expanded in `glsl_shaders.cpp` as well as `#extension` being unconditionally expanded. That would avoid the complexity of needing to preprocess but it would still allow per-platform extensions.